chef/data_bags/crowbar/template-crowbar.json

```diff
@@ -28,10 +28,10 @@
     "crowbar": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 11,
+      "schema-revision": 100,
       "element_states": {
         "crowbar": [ "all" ],
-        "crowbar-upgrade": [ "crowbar_upgrade" ]
+        "crowbar-upgrade": [ "readying", "ready", "applying", "crowbar_upgrade" ]
```

Comment:
Ok. Just so you know: I'd like to remove most of the roles running in readying/ready at some point ;-) But this can be done later on indeed.